compose/composer.go

```diff
@@ -122,6 +126,20 @@ func (c *Composer) Apply() {
 	logger.Log.Infoln("Successfully Completed!!")
 }
 
+func (c *Composer) PrintConfig() {
+	fmt.Println("##########################")
+	fmt.Println("# Compose Configuration #")
+	fmt.Println("##########################")
+	fmt.Printf("ConfigFile: %s\n", c.ConfigFile)
+	if c.CherryPickedFlow != "" {
+		fmt.Printf("Selected Flow: %s\n", c.CherryPickedFlow)
+	}
+	fmt.Printf("DryRun: %t\n", c.DryRun)
+	fmt.Printf("LogLevel: %s\n", c.LogLevel)
+	fmt.Printf("WorkersCount: %d\n", c.WorkersCount)
```

Comment:
@pandeykalyan-cf  I think we need to refactor this one by just printing the effective worker count. If I supply `100` as worker count then it will just say `100` rather than overriding with the number of flows count.